api/provider/aws/common_test.go

```diff
@@ -3,11 +3,96 @@ package aws
 import (
 	"testing"
 
+	"github.com/aws/aws-sdk-go/service/ecs"
 	"github.com/quintilesims/layer0/api/tag"
 	"github.com/quintilesims/layer0/common/models"
 	"github.com/stretchr/testify/assert"
 )
 
+func Test_getLaunchTypeFromEnvironmentID(t *testing.T) {
+	tagStore := tag.NewMemoryStore()
+
+	envIDs := []string{"env_id0", "env_id1"}
+
+	tags := models.Tags{
+		{
+			EntityID:   envIDs[0],
```

Comment:
I'm not sure we do this pattern anywhere else. Maybe just use 
`env_id1, env_id2`
[example](https://github.com/quintilesims/layer0/blob/api-refactor/api/provider/aws/test_aws/environment_list_test.go#L28)

Reply:
@nehayward Yeah, I also don't think we do the pattern anywhere else. I introduced it looking for some feedback on whether we like this micropattern; I can see it not providing a benefit in `Test_getLaunchTypeFromEnvironmentID()` beyond a mild maintenance streamline wherein updates only have to be done in one place (and maybe that's benefit enough to merit keeping the pattern?), but I _do_ think there's a good reason for the micropattern in `Test_getLaunchTypeFromEnvironmentID_errors()` when we `range` over the `envIDs`.

Happy to strip it out if it's overengineering, though. :)

Reply:
I think it makes sense in the `Test_getLaunchTypeFromEnvironmentID_Errors` but we should revert to the old pattern for consistency in Test_getLaunchTypeFromEnvironmentID. Just my opinion though.

Reply:
I like it. ✅